kornia/geometry/liegroup/se3.py

```diff
@@ -248,6 +248,24 @@ def inverse(self) -> 'Se3':
         r_inv = self.r.inverse()
         return Se3(r_inv, r_inv * (-1 * self.t))
 
+    @classmethod
+    def random(cls, batch_size: Optional[int] = None, device=None, dtype=None) -> 'Se3':
```

Comment:
what do you think about instead of batch_size: Optional[int]` pass a `shape: Optional[Tuple[int, ...]]` ?
as i propose here: https://github.com/kornia/kornia/pull/1987/files#diff-42adf0d69e7dd1e765e13b264d372bd6aa41fd7e2e274edf2ce76ba05a6bb82fR44

not sure if makes sense here in this apis though

Reply:
i think introducing shape here can be a bit confusing since its a lie group